batch7crud-roll2/src/main/webapp/css/style.css

```diff
@@ -0,0 +1,305 @@
+body, html {
+    height: 100%;
+    position: relative;
+}
+
+.mainWrapper {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename selector mainWrapper to match the regular expression: ^[a-z][a-z\-]*$ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Aselector-naming-convention)